README.md

```diff
@@ -29,7 +29,8 @@ A beautiful blogging platform for the JankariTech peoples.
     authorName: John Doe
     authorAvatar: https://some.link.jpg (must be an image URL, **required**)
     authorLink: https://github.com/John
-    createdAt: Oct 30, 2019 (must be in the format `MM dd, yyyy`, **required**)
+    createdAt: Oct 30, 2019 (can be in the format `MMM dd, yyyy` or `MMMM dd, yyyy`, **required**)
+    updatedAt: October 30, 2023 (can be in the format `MMM dd, yyyy` or `MMMM dd, yyyy`, **optional**)
```

Comment:
are you using on purpose a different format than in the example above to demonstrate that both date formats are supported? 
I think it would be clearer that the date has been updated if both lines would use the same date format in this example

Reply:
The format `MM dd, yyyy` was wrong for `Oct 30, 2019` in the previous example. Our blog works for any format `MM dd, yyyy` or `MMM dd, yyyy` or `MMMM dd, yyyy`. But, the first one might create confusion so I mentioned only two formats are supported.

Reply:
Yeah, my comment was about the two examples given:
- Oct 30, 2019
- October 30, 2023

Did you on purpose use different format to demonstrate that both formats are valid? I think it would read nicer and show better the difference between `createdAt` and `updatedAt` if you use the same format in the documetation

Reply:
I was trying to provide an example for both supported formats. If this approach is confusing, can we add an example for each format in the description?
```
createdAt: Oct 30, 2019 (must be in the format `MMM dd, yyyy` e.g., `Oct 30, 2019` or `MMMM dd, yyyy` e.g., `October 30, 2019`, **required**)
updatedAt: Oct 30, 2019 (must be in the format `MMM dd, yyyy` e.g., `Oct 30, 2019` or `MMMM dd, yyyy` e.g., `October 30, 2019`, **optional**)
```

Reply:
If it's on purpose, that's fine. 

And your suggestion is also good - having different year (2019 and 2023) like in your original suggestion might be helpful for understanding

```
createdAt: Oct 30, 2019 (must be in the format `MMM dd, yyyy` e.g., `Oct 30, 2019` or `MMMM dd, yyyy` e.g., `October 30, 2019`, **required**)
updatedAt: Oct 30, 2023 (must be in the format `MMM dd, yyyy` e.g., `Oct 30, 2023` or `MMMM dd, yyyy` e.g., `October 30, 2023`, **optional**)
```